src/tile.js

```diff
@@ -154,11 +154,12 @@ $.Tile = function(level, x, y, bounds, exists, url, context2D) {
      */
     this.opacity    = null;
     /**
-     * The distance of this tile to the viewport center.
-     * @member {Number} distance
+     * The squared distance of this tile to the viewport center.
+     * Use for comparing tiles.
+     * @member {Number} squaredDistance
      * @memberof OpenSeadragon.Tile#
      */
-    this.distance   = null;
+    this.squaredDistance   = null;
```

Comment:
According to @avandecreme ([here](https://github.com/openseadragon/openseadragon/pull/1006/files/f0cb707ff238c0c9fcc7963524beff8797dc3e83..2e3f57401fa9eea094112fa0c317e26fdaaf2a42#r77874711)) `@private` doc comments are no longer shown on the public website... please just add a `@private` here.